src/view/configuration/advertisingSection.jsx

```diff
@@ -420,8 +420,7 @@ const AdvertisingSection = ({ instanceFieldName, initInfo }) => {
           response?.items || response?.data || response || [];
 
         setAdvertisers(advertisersList);
-      } catch (e) {
-        console.error("Failed to fetch advertisers:", e);
+      } catch {
```

Comment:
I removed this because the error was polluting the test report. I checked and in no other place we console.error any api failure.